net-analyzer/driftnet/driftnet-1.6.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools fcaps
+
+DESCRIPTION="Watches network traffic and displays media from TCP streams observed"
+HOMEPAGE="https://chris.ex-parrot.com/driftnet/"
+SRC_URI="https://github.com/deiv/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc -sparc ~x86"
+IUSE="debug gtk httpd suid test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	net-libs/libpcap
+	media-libs/libwebp:=
+	gtk? (
+		dev-libs/glib:2
+		media-libs/giflib:=
+		media-libs/libjpeg-turbo:=
+		media-libs/libpng:=
+		x11-libs/cairo
+		x11-libs/gtk+:3[X]
+	)
+	httpd? ( net-libs/libwebsockets:=[client,http-proxy,socks5] )
+"
+DEPEND="
+	${RDEPEND}
+	test? ( dev-util/cmocka )
+"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	# https://github.com/deiv/driftnet/pull/56.patch
+	"${FILESDIR}"/${PN}-1.6.0-pr56.patch
```

Comment:
The naming is not suitable, see https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html
Because it's just rebased from existing patches for 1.5.0, use the same name seems better.